hack/run-must-gather.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e
+
+# expect oc to be in PATH by default
+OC_TOOL="${OC_TOOL:-oc}"
+# Current Mustgather version
+VERSION="4.9-snapshot"
+# Must gather image url
+MUST_GATHER_IMG="quay.io/openshift-kni/performance-addon-operator-must-gather:$VERSION"
+OUTPUT_DIR="/tmp/mustgather-$VERSION"
```

Comment:
maybe it is better to use `/tmp/mustgather-$VERSION-$timestamp`

Reply:
@cynepco3hahue okay will make the changes, but does the system on which this directory exists persist for subsequent runs ?,  Because if i am looking a way to pass this directory to another script which will run Performance profile creator to generate the profile.

Reply:
I am thinking about the case when you will want to test it locally, or under some persistent environment.